tests/test_config_entries.py

```diff
@@ -5547,6 +5531,80 @@ async def mock_setup_entry_platform(
     )
 
 
+async def test_config_entry_unloaded_during_platform_setups(
```

Comment:
Are we sure this test won't be flaky? If we're not sure, it might be better to use a bunch of `asyncio.Event`-objects to make the execution fully deterministic.

Reply:
I don't think it will, but no harm in making it safer.

done in cc58e86f3767c4f5fed94b0683fd7b3173692baa